code/style.css

```diff
@@ -1,4 +1,213 @@
+
+/* Universal styles */
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
+  font-family: 'Work sans', Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+  background-color: #9DC88D;
+}
+
+h1, h2 {
+  font-family: 'Neuton', Garamond, serif;
+}
+
+p {
+  font-family: 'Work sans', Helvetica, sans-serif;
+  line-height: 1.4;
+}
+
+a {
+  font-family: 'Work sans', Helvetica, sans-serif;
+  font-weight: 700;
+}
+
+/* Header content */
+
+.header {
+  height: 5rem;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 1rem 0;
+  background-color: #164A41;
+  color: #FFFFFF;
+}
+
+.logo-container {
+  padding: 1rem;
+}
+
+.header h1 {
+  font-size: 2.5rem;
+}
+
+.header-nav {
+  display: none;
+  /* The nav is not displayed in mobile devices*/
+}
+
+.header-nav li {
+  display: inline;
+}
+
+.header-nav a {
+  text-decoration: none;
+  letter-spacing: 0.08rem;
+  padding: 0 0.5rem;
+  color: #F1B24A;
+}
+
+.header-nav a:hover {
+  text-decoration: underline;
+}
+
+.main-container {
+  max-width: 1000px;
+  margin: 0 auto;
+  /*Centers the main container*/
+}
+
+/* Big news section */
+
+.big-news-section {
+  display: grid;
+  grid-template-rows: 1fr auto;
+  grid-template-columns: 1fr;
```

Comment:
Is it possible to get the same result by only using column for the mobile version? In my mind I think this just gives one cell if there is one column and one row,

Reply:
I'm not sure but I'll try to find out if it's possible.